src/styles/blocks/calendar.scss

```diff
@@ -1,52 +1,52 @@
 .calendar {
-  display: flex;
-  flex-wrap: wrap;
   position: relative;
   top: 50%;
-  transform: translateY(-50%);
+  display: flex;
+  flex-wrap: wrap;
   margin: auto;
```

Comment:
Don't set margins to block itself. You could add styles to the parent element to center this block.